### PR TITLE
Fix the order of results

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Fixes https://github.com/ooni/probe/issues/1162

## Proposed Changes

  - Add fixed sorting algorithm

### CODE REVIEW

File: ooniprobe/Model/Database/Result.mm [hunk 1]
```diff
@@ -18,14 +18,37 @@ - (SRKResultSet*)measurements {
...
+    for (NSString *testName in testOrder){
+        for (Measurement *current in measurements){
+            if ([current.test_name isEqualToString:testName]){
+                [measurementsSorted addObject:current];
+                break;
+            }
+        }
+    }
+    return [[SRKResultSet alloc] initWithArray:measurementsSorted];
+    //return measurementsSorted;
```
Comment (bassosimone) [inline ooniprobe/Model/Database/Result.mm:51]: please remove this commented out line

File: ooniprobe/Model/Database/Result.mm [hunk 2]
```diff
@@ -18,14 +18,37 @@ - (SRKResultSet*)measurements {
...
+                   where:@"result_id = ? AND is_rerun = 0 AND is_done = 1"
+                   parameters:@[self]]
+                   orderByDescending:@"is_anomaly"]
+                   orderByDescending:@"is_failed"]
+                   order:@"Id"]
+                   fetch];
+    }
+    NSMutableArray *measurementsSorted = [NSMutableArray new];
+    SRKResultSet *measurements = self.measurements;
+    NSArray *testOrder = [[TestUtility getTests] objectForKey:self.test_group_name];
```
Comment (bassosimone) [inline ooniprobe/Model/Database/Result.mm:39]: where's the testOrder? is it already in master?

File: ooniprobe/Model/Database/Result.mm (hunk 1 above)
Comment (lorenzoPrimi) [reply ooniprobe/Model/Database/Result.mm:51]: done

File: ooniprobe/Model/Database/Result.mm (hunk 2 above)
Comment (lorenzoPrimi) [reply ooniprobe/Model/Database/Result.mm:39]: yes, it's in the `TestUtility` class

**Pull request merged**